docs/xks/developer-guide/security.md

```diff
@@ -51,6 +51,29 @@ For example:
           runAsUser: 1000
 ```
 
- The `runAsUser` field specifies that for any Containers in the Pod, all processes run with user ID 1000. The runAsGroup field specifies the primary group ID of 1000 for all processes within any containers of the Pod
+ The `runAsUser` field specifies that for any Containers in the Pod, all processes run with user ID 1000. The runAsGroup field specifies the primary group ID of 1000 for all processes within any containers of the Pod. If this field is not set, the primary group ID of the containers will be root. Using `runAsNonRoot: true` makes it impossible for containers to run as root, it will require a defined non-zero numeric USER directive defined in the container image.
 
-You can read more in the [official documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+You can read more in the official documentation on [Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+ and [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
```

Comment:
No need to talk about PSP. We don't use it in general since we use OPA instead and it will be deprecated in version 1.25.